app/javascript/packages/document-capture/components/in-person-location-post-office-search-step.tsx

```diff
@@ -201,12 +199,14 @@ function InPersonLocationPostOfficeSearchStep({ onChange, toPreviousStep, regist
   return (
     <>
       <PageHeading>{t('in_person_proofing.headings.location')}</PageHeading>
-      {arcgisSearchEnabled && (
+      <div aria-live="polite">
         <AddressSearch onAddressFound={handleFoundAddress} registerField={registerField} />
-      )}
-      <p>{t('in_person_proofing.body.location.location_step_about')}</p>
-      {locationsContent}
-      <BackButton onClick={toPreviousStep} />
+      </div>
+      <div role="status">
+        <p>{t('in_person_proofing.body.location.location_step_about')}</p>
+        {locationsContent}
+        <BackButton onClick={toPreviousStep} />
```

Comment:
another thing happening with the screen reader...it's currently reading the text that appears alongside the search button after it reads out "back"

Reply:
if that address is removed then this screen reader issue should be ignored.